lib/constants.ts

```diff
@@ -36,7 +36,28 @@ export const INCOMING_SOUND_URL: string =
  * @private
  * Test names.
  */
-export enum TestNames {
+export enum TestName {
   InputAudioDevice = 'input-volume',
   OutputAudioDevice = 'output-volume',
 }
+
+/**
+ * All of the expected error names to be thrown by the diagnostics tests.
+ * These names are set in the error objects as the `.name` member.
+ */
+export enum ErrorName {
+  AlreadyStoppedError = 'already-stopped',
+  DiagnosticError = 'diagnostic',
+  InvalidOptionError = 'invalid-option',
+  InvalidOptionsError = 'invalid-options',
+  InvalidStateError = 'invalid-state',
+  PromiseTimedOutError = 'promise-timed-out',
+  UnsupportedError = 'unsupported',
+}
+
+/**
+ * All of the expected warnings to be thrown by the diagnostics tests.
+ */
+export enum WarningName {
```

Comment:
I like the idea of `Warning` here over `WarningName` because we don't have classes for each warning; the warning is the `string` itself, there's no extra data. But I would also like to keep this one consistent with the decision we make for `Error`/`ErrorName`. I'm not sure on using just `Error` though because the enum doesn't contain the actual error classes, just the names. WDYT @ryan-rowland?